packages/capabilities/test/capabilities/provider.test.js

```diff
@@ -0,0 +1,68 @@
+import assert from 'assert'
+import { access } from '@ucanto/validator'
+import { Verifier } from '@ucanto/principal/ed25519'
+import * as Provider from '../../src/provider.js'
+import { bob, service, mallory } from '../helpers/fixtures.js'
+import * as ucanto from '@ucanto/core'
+
+describe('provider/add', function () {
+  it.skip('can invoke as account with ./update', async function () {
```

Comment:
This does not work because it requires `access/authorize` thing as you did in other tests.

Reply:
fixed in #477 (into this branch) I think